packages/components/src/components/domainproperties/dataset/constants.ts

```diff
@@ -0,0 +1,19 @@
+
+export const DATASET_NAME_TIP = "This name must be unique. It is used when identifying datasets during data upload.";
+
+export const DATASET_CATEGORY_TIP = "Assigning a category to a dataset will group it with similar datasets in the navigator and data browser.";
+
+export const DATASET_LABEL_TIP = "The name of the dataset shown to users. If no Label is provided, the Name is used.";
+
+export const DATASET_ID_TIP = "The unique, numerical identifier for your dataset. It is defined during dataset creation and cannot be modified.";
+
+export const VISIT_DATE_TIP = "If the official 'Visit Date' for a visit can come from this dataset, choose the date column to represent it. Note that since datasets can include data from many visits, each visit must also indicate the official 'VisitDate' dataset.";
+
+export const COHORT_TIP =  "Datasets may be cohort specific, or associated with all cohorts.";
+
+export const TAG_TIP = "Adding a tag provides an additional, flexible way to categorize this dataset.";
+
+// TODO: add tip
+export const DATASPACE_TIP = "Coming soon...";
```

Comment:
See this doc page for help with the tooltip text: https://www.labkey.org/Documentation/wiki-page.view?name=shareddatasets